packages/core/ui/theme.js

```diff
@@ -1,34 +1,40 @@
 import { createMuiTheme } from '@material-ui/core/styles'
 import { grey } from '@material-ui/core/colors'
 
+const midnight = '#0D233F'
+const grape = '#721E63'
+const forest = '#135560'
+const mandarin = '#FFB11D'
+
 export default createMuiTheme({
   palette: {
     // type: 'dark',
-    primary: { main: '#0D233F' },
-    secondary: { main: '#721E63' },
+    primary: { main: midnight },
+    secondary: { main: grape },
+    tertiary: { main: forest },
```

Comment:
The `primary` and `secondary` entries are special because they use the `main` to calculate other values for that color. I think we can simulate it for `tertiary` with something like this:

```suggestion
    tertiary: {
      light: lighten(forest, 0.2),
      main: forest,
      dark: darken(forest, 0.3),
      contrastText:
        getContrastRatio(forest, '#fff') >= 3 ? '#fff' : 'rgba(0, 0, 0, 0.87)',
    },
```

where the import for those util functions is

```js
import {
  lighten,
  darken,
  getContrastRatio,
} from '@material-ui/core/styles/colorManipulator'
```

Reply:
good idea, done